<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="bsafe_app" timestamp="2024-08-08T19:22:19Z" hostname="f5d6db3fc801" tests="2" skipped="0" failures="0" errors="2" time="33.763">
    <testcase time="17.147" classname="bsafe_app" name="bsafe_module_starts">
      <error type="error" message="Error in `app_initialize(self, private, app_dir = app_dir, ..., load_timeout = load_timeout,  ('test-bsafe_app.R:4:5')">Error in `app_initialize(self, private, app_dir = app_dir, ..., load_timeout = load_timeout, 
    timeout = timeout, wait = wait, expect_values_screenshot_args = expect_values_screenshot_args, 
    screenshot_args = screenshot_args, check_names = check_names, 
    name = name, variant = variant, view = view, height = height, 
    width = width, seed = seed, clean_logs = clean_logs, shiny_args = shiny_args, 
    render_args = render_args, options = options)`: Shiny app did not become stable in 15000ms.
Message: Timed out waiting for JavaScript script to return `true`
ℹ `timeout`: 15000
ℹ `script`:
window.shinytest2 &amp;&amp; window.shinytest2.ready === true
Caused by error in `chromote_wait_for_condition()`:
! Timed out waiting for JavaScript script to return `true`
ℹ `timeout`: 15000
ℹ `script`:
window.shinytest2 &amp;&amp; window.shinytest2.ready === true


ℹ You can inspect the failed AppDriver object via `rlang::last_error()$app`
ℹ AppDriver logs:
{shinytest2} R  info   19:22:20.24 Start AppDriver initialization
{shinytest2} R  info   19:22:20.28 Starting Shiny app
{shinytest2} R  info   19:22:21.12 Creating new ChromoteSession
{shinytest2} R  info   19:22:21.19 Navigating to Shiny app
{shinytest2} R  info   19:22:21.29 Injecting shiny-tracer.js
{chromote}   JS info   19:22:21.31 shinytest2; jQuery not found
{chromote}   JS info   19:22:21.32 shinytest2; Loaded
{shinytest2} R  info   19:22:21.32 Waiting for Shiny to become ready
{shinytest2} R  info   19:22:36.37 Error while initializing AppDriver:
                                   Shiny app did not become stable in 15000ms.
                                   Message: Timed out waiting for JavaScript script to return `true`
                                   ℹ `timeout`: 15000
                                   ℹ `script`:
                                   window.shinytest2 &amp;&amp; window.shinytest2.ready === true
                                   Caused by error in `chromote_wait_for_condition()`:
                                   ! Timed out waiting for JavaScript script to return `true`
                                   ℹ `timeout`: 15000
                                   ℹ `script`:
                                   window.shinytest2 &amp;&amp; window.shinytest2.ready === true
{shiny}      R  stderr ----------- Loading required package: shiny
{shiny}      R  stderr ----------- Running application in test mode.
{shiny}      R  stderr ----------- 
{shiny}      R  stderr ----------- Listening on http://127.0.0.1:7936
{shiny}      R  stderr ----------- Warning: Error in mod_simulation_ui: object 'ui' not found
{shiny}      R  stderr -----------   80: mod_simulation_ui
{shiny}      R  stderr -----------   79: bsafe_UI
{shiny}      R  stderr -----------   73: ui
{shiny}      R  stderr -----------   14: &lt;Anonymous&gt;
{shiny}      R  stderr -----------   12: &lt;Anonymous&gt;
{shiny}      R  stderr -----------   11: &lt;Anonymous&gt;
{shiny}      R  stderr -----------   10: base::do.call
{shiny}      R  stderr -----------    9: base::saveRDS
{shiny}      R  stderr -----------    8: base::withCallingHandlers
{shiny}      R  stderr -----------    7: doTryCatch
{shiny}      R  stderr -----------    6: tryCatchOne
{shiny}      R  stderr -----------    5: tryCatchList
{shiny}      R  stderr -----------    2: tryCatchList
{shiny}      R  stderr -----------    1: base::tryCatch


Caused by error in `app_initialize_()`:
! Shiny app did not become stable in 15000ms.
Message: Timed out waiting for JavaScript script to return `true`
ℹ `timeout`: 15000
ℹ `script`:
window.shinytest2 &amp;&amp; window.shinytest2.ready === true
Caused by error in `chromote_wait_for_condition()`:
! Timed out waiting for JavaScript script to return `true`
ℹ `timeout`: 15000
ℹ `script`:
window.shinytest2 &amp;&amp; window.shinytest2.ready === true
Backtrace:
     ▆
  1. ├─shinytest2::AppDriver$new(teal.modules.bsafe:::mock_bsafe()) at test-bsafe_app.R:4:5
  2. │ └─shinytest2 (local) initialize(...)
  3. │   └─shinytest2:::app_initialize(...)
  4. │     ├─base::withCallingHandlers(...)
  5. │     └─shinytest2:::app_initialize_(self, private, ..., view = view)
  6. │       ├─base::withCallingHandlers(...)
  7. │       └─self$wait_for_js(...)
  8. │         └─shinytest2:::app_wait_for_js(...)
  9. │           └─shinytest2:::chromote_wait_for_condition(...)
 10. │             └─rlang::abort(...)
 11. │               └─rlang:::signal_abort(cnd, .file)
 12. │                 └─base::signalCondition(cnd)
 13. ├─shinytest2 (local) `&lt;fn&gt;`(`&lt;rlng_rrr&gt;`)
 14. │ └─shinytest2:::app_abort(...)
 15. │   └─rlang::abort(..., app = self, call = call)
 16. │     └─rlang:::signal_abort(cnd, .file)
 17. │       └─base::signalCondition(cnd)
 18. └─shinytest2 (local) `&lt;fn&gt;`(`&lt;rlng_rrr&gt;`)
 19.   └─shinytest2:::app_abort(...)
 20.     └─rlang::abort(..., app = self, call = call)</error>
    </testcase>
    <testcase time="16.616" classname="bsafe_app" name="tm_bsafe_starts">
      <error type="error" message="Error in `app_initialize(self, private, app_dir = app_dir, ..., load_timeout = load_timeout,  ('test-bsafe_app.R:13:5')">Error in `app_initialize(self, private, app_dir = app_dir, ..., load_timeout = load_timeout, 
    timeout = timeout, wait = wait, expect_values_screenshot_args = expect_values_screenshot_args, 
    screenshot_args = screenshot_args, check_names = check_names, 
    name = name, variant = variant, view = view, height = height, 
    width = width, seed = seed, clean_logs = clean_logs, shiny_args = shiny_args, 
    render_args = render_args, options = options)`: Shiny app did not become stable in 15000ms.
Message: An error occurred while waiting for Shiny to be stable
Caused by error in `app_wait_for_idle()`:
! An error occurred while waiting for Shiny to be stable


ℹ You can inspect the failed AppDriver object via `rlang::last_error()$app`
ℹ AppDriver logs:
{shinytest2} R  info   19:22:36.59 Start AppDriver initialization
{shinytest2} R  info   19:22:36.78 Starting Shiny app
{shinytest2} R  info   19:22:37.60 Creating new ChromoteSession
{shinytest2} R  info   19:22:37.65 Navigating to Shiny app
{shinytest2} R  info   19:22:37.72 Injecting shiny-tracer.js
{chromote}   JS info   19:22:37.73 shinytest2; jQuery not found
{chromote}   JS info   19:22:37.73 shinytest2; Loaded
{shinytest2} R  info   19:22:37.73 Waiting for Shiny to become ready
{chromote}   JS info   19:22:37.82 shinytest2; jQuery found
{chromote}   JS info   19:22:37.82 shinytest2; Waiting for shiny session to connect
{chromote}   JS info   19:22:37.88 shinytest2; Connected
{chromote}   JS info   19:22:37.92 shinytest2; shiny:busy
{shinytest2} R  info   19:22:37.94 Waiting for Shiny to become idle for 200ms within 15000ms
{chromote}   JS info   19:22:37.95 shinytest2; Waiting for Shiny to be stable
{shinytest2} R  info   19:22:52.96 Error while initializing AppDriver:
                                   Shiny app did not become stable in 15000ms.
                                   Message: An error occurred while waiting for Shiny to be stable
                                   Caused by error in `app_wait_for_idle()`:
                                   ! An error occurred while waiting for Shiny to be stable
{shiny}      R  stderr ----------- Loading required package: shiny
{shiny}      R  stderr ----------- Attempting to load the environment ‘package:teal.modules.bsafe’
{shiny}      R  stderr ----------- Loading required package: teal.data
{shiny}      R  stderr ----------- Loading required package: teal.code
{shiny}      R  stderr ----------- Loading required package: teal.slice
{shiny}      R  stderr ----------- 
{shiny}      R  stderr ----------- You are using teal version 0.15.2
{shiny}      R  stderr ----------- 
{shiny}      R  stderr ----------- Attaching package: ‘teal’
{shiny}      R  stderr ----------- 
{shiny}      R  stderr ----------- The following objects are masked from ‘package:teal.slice’:
{shiny}      R  stderr ----------- 
{shiny}      R  stderr -----------     as.teal_slices, teal_slices
{shiny}      R  stderr ----------- 
{shiny}      R  stderr ----------- Running application in test mode.
{shiny}      R  stderr ----------- 
{shiny}      R  stderr ----------- Listening on http://127.0.0.1:6575
{shiny}      R  stderr ----------- Warning: Error in mod_simulation_ui: object 'ui' not found
{shiny}      R  stderr -----------   14: &lt;Anonymous&gt;
{shiny}      R  stderr -----------   12: &lt;Anonymous&gt;
{shiny}      R  stderr -----------   11: &lt;Anonymous&gt;
{shiny}      R  stderr -----------   10: base::do.call
{shiny}      R  stderr -----------    9: base::saveRDS
{shiny}      R  stderr -----------    8: base::withCallingHandlers
{shiny}      R  stderr -----------    7: doTryCatch
{shiny}      R  stderr -----------    6: tryCatchOne
{shiny}      R  stderr -----------    5: tryCatchList
{shiny}      R  stderr -----------    2: tryCatchList
{shiny}      R  stderr -----------    1: base::tryCatch
{shiny}      R  stdout ----------- [INFO] 2024-08-08 19:22:38.0741 pid:4645 token:[6578c6f2] teal Initializing reporter_previewer_module


Caused by error in `app_initialize_()`:
! Shiny app did not become stable in 15000ms.
Message: An error occurred while waiting for Shiny to be stable
Caused by error in `app_wait_for_idle()`:
! An error occurred while waiting for Shiny to be stable
Backtrace:
     ▆
  1. ├─base::suppressWarnings(shinytest2::AppDriver$new(teal.modules.bsafe:::mock_teal())) at test-bsafe_app.R:13:5
  2. │ └─base::withCallingHandlers(...)
  3. ├─shinytest2::AppDriver$new(teal.modules.bsafe:::mock_teal())
  4. │ └─shinytest2 (local) initialize(...)
  5. │   └─shinytest2:::app_initialize(...)
  6. │     ├─base::withCallingHandlers(...)
  7. │     └─shinytest2:::app_initialize_(self, private, ..., view = view)
  8. │       ├─base::withCallingHandlers(...)
  9. │       └─self$wait_for_idle(duration = 200, timeout = private$load_timeout)
 10. │         └─shinytest2:::app_wait_for_idle(...)
 11. │           └─shinytest2:::app_abort(self, private, "An error occurred while waiting for Shiny to be stable")
 12. │             └─rlang::abort(..., app = self, call = call)
 13. │               └─rlang:::signal_abort(cnd, .file)
 14. │                 └─base::signalCondition(cnd)
 15. ├─shinytest2 (local) `&lt;fn&gt;`(`&lt;rlng_rrr&gt;`)
 16. │ └─shinytest2:::app_abort(...)
 17. │   └─rlang::abort(..., app = self, call = call)
 18. │     └─rlang:::signal_abort(cnd, .file)
 19. │       └─base::signalCondition(cnd)
 20. └─shinytest2 (local) `&lt;fn&gt;`(`&lt;rlng_rrr&gt;`)
 21.   └─shinytest2:::app_abort(...)
 22.     └─rlang::abort(..., app = self, call = call)</error>
    </testcase>
  </testsuite>
  <testsuite name="decision_making" timestamp="2024-08-08T19:22:53Z" hostname="f5d6db3fc801" tests="16" skipped="0" failures="2" errors="1" time="32.085">
    <testcase time="0.0730000000000004" classname="decision_making" name="calc_log_hazard_area_returns_correct_values"/>
    <testcase time="0.00300000000000011" classname="decision_making" name="calc_param_approx_boundaries_returns_correct_lower_and_upper_boundaries"/>
    <testcase time="12.48" classname="decision_making" name="can_read_and_change_distribution"/>
    <testcase time="1.92899999999999" classname="decision_making" name="can_read_and_change_non_ae_perc_slider"/>
    <testcase time="0.00900000000000034" classname="decision_making" name="ae_div_is_hidden_when_analysis_type_is_BSAFE_CHOICES_SEL_ANALYSIS_1_"/>
    <testcase time="0.00800000000000267" classname="decision_making" name="inci_div_is_visible_when_analysis_type_is_BSAFE_CHOICES_SEL_ANALYSIS_1_"/>
    <testcase time="0.956000000000003" classname="decision_making" name="header_element_is_present"/>
    <testcase time="0.919999999999995" classname="decision_making" name="preface_element_is_present"/>
    <testcase time="0.941000000000003" classname="decision_making" name="plot_is_present"/>
    <testcase time="1.208" classname="decision_making" name="auc_text_element_is_present">
      <failure type="failure" message="Check on 'auc_text' failed: Must be of type 'string', not 'list' ('test-decision_making.R:76:5')">Check on 'auc_text' failed: Must be of type 'string', not 'list'
Backtrace:
    ▆
 1. └─checkmate::expect_string(auc_text, min.chars = 1, null.ok = FALSE) at test-decision_making.R:76:5
 2.   └─checkmate::makeExpectation(x, res, info, label)</failure>
    </testcase>
    <testcase time="0.945" classname="decision_making" name="preset_statements_table_is_present">
      <failure type="failure" message="`opening_tag` has length 0, not length 1. ('test-decision_making.R:82:5')">`opening_tag` has length 0, not length 1.</failure>
    </testcase>
    <testcase time="0.971000000000004" classname="decision_making" name="exported_preset_statements_code_matches_output">
      <error type="error" message="Error in `bsafe::mix_distribution_all(current_trial_data = current_trial_data(),  ('test-decision_making.R:86:5')">Error in `bsafe::mix_distribution_all(current_trial_data = current_trial_data(), 
    select_dist = input[[BSAFE_ID$SEL_DIST]], select_analysis = analysis_type(), 
    param_approx = param_approx(), robust_map_object = robust_map_mcmc(), 
    post_dist = post_dist(), seed = seed())`: unused argument (seed = seed())
Backtrace:
     ▆
  1. ├─shiny::isolate(app$get_values()[["export"]][["r"]][["preset_statements"]]()) at test-decision_making.R:86:5
  2. │ ├─shiny::..stacktraceoff..(...)
  3. │ └─ctx$run(...)
  4. │   ├─promises::with_promise_domain(...)
  5. │   │ └─domain$wrapSync(expr)
  6. │   ├─shiny::withReactiveDomain(...)
  7. │   │ └─promises::with_promise_domain(...)
  8. │   │   └─domain$wrapSync(expr)
  9. │   │     └─base::force(expr)
 10. │   └─env$runWith(self, func)
 11. │     └─shiny (local) contextFunc()
 12. │       └─shiny::..stacktraceon..(expr)
 13. └─app$get_values()[["export"]][["r"]][["preset_statements"]]()
 14.   ├─shinymeta:::metaDispatch(...)
 15.   │ ├─shinymeta:::switchMetaMode(...)
 16.   │ └─base::force(normal)
 17.   └─shiny (local) r_normal()</error>
    </testcase>
    <testcase time="2.36" classname="decision_making" name="exported_plot_code_matches_output"/>
    <testcase time="9.265" classname="decision_making" name="can_read_and_change_ae_perc_slider"/>
    <testcase time="0.00900000000000034" classname="decision_making" name="ae_div_is_hidden_when_analysis_type_is_BSAFE_CHOICES_SEL_ANALYSIS_2_"/>
    <testcase time="0.00799999999999557" classname="decision_making" name="inci_div_is_visible_when_analysis_type_is_BSAFE_CHOICES_SEL_ANALYSIS_2_"/>
  </testsuite>
  <testsuite name="map_prior" timestamp="2024-08-08T19:23:25Z" hostname="f5d6db3fc801" tests="27" skipped="0" failures="0" errors="1" time="76.179">
    <testcase time="0.311999999999998" classname="map_prior" name="preface_prior_txt_returns_correct_preface_for_MAP_approach"/>
    <testcase time="0.00400000000000489" classname="map_prior" name="preface_prior_txt_returns_correct_preface_for_log_scale_approach"/>
    <testcase time="0.007000000000005" classname="map_prior" name="preface_prior_txt_returns_an_error_for_unknown_selection_analysis"/>
    <testcase time="10.645" classname="map_prior" name="map_prior_is_calculated_when_button_is_pressed">
      <error type="error" message="Error in `app_wait_for_idle(self, private, duration = duration, timeout = timeout)`: An error occurred while waiting for Shiny to be stable ('test-map_prior.R:38:5')">Error in `app_wait_for_idle(self, private, duration = duration, timeout = timeout)`: An error occurred while waiting for Shiny to be stable
Backtrace:
    ▆
 1. └─app$wait_for_idle() at test-map_prior.R:38:5
 2.   └─shinytest2:::app_wait_for_idle(self, private, duration = duration, timeout = timeout)
 3.     └─shinytest2:::app_abort(self, private, "An error occurred while waiting for Shiny to be stable")
 4.       └─rlang::abort(..., app = self, call = call)</error>
    </testcase>
    <testcase time="1.545" classname="map_prior" name="forest_plot_is_present"/>
    <testcase time="1.41499999999999" classname="map_prior" name="mix_density_plot_is_present"/>
    <testcase time="1.435" classname="map_prior" name="preface_prior_text_is_present"/>
    <testcase time="1.419" classname="map_prior" name="density_function_is_present"/>
    <testcase time="1.486" classname="map_prior" name="summary_table_is_present"/>
    <testcase time="1.55" classname="map_prior" name="map_mcmc_returned_values_is_a_gMAP_object"/>
    <testcase time="1.435" classname="map_prior" name="param_approx_returned_values_is_a_EM_object"/>
    <testcase time="1.48099999999999" classname="map_prior" name="adj_tau_returned_values_is_a_numeric"/>
    <testcase time="1.748" classname="map_prior" name="ess_method_is_a_subset_from_BSAFE_CHOICES_SEL_ESS_METHOD"/>
    <testcase time="1.432" classname="map_prior" name="forest_plot_is_a_ggplot"/>
    <testcase time="1.452" classname="map_prior" name="map_summary_table_is_a_data.frame"/>
    <testcase time="5.401" classname="map_prior" name="map_mcmc_code_eval_matches_print"/>
    <testcase time="5.746" classname="map_prior" name="param_approx_code_eval_matches_print"/>
    <testcase time="1.41000000000001" classname="map_prior" name="adj_tau_code_eval_matches_value"/>
    <testcase time="5.514" classname="map_prior" name="forest_plot_code_eval_matches_plot"/>
    <testcase time="5.438" classname="map_prior" name="map_summary_table_code_eval_matches_value"/>
    <testcase time="5.932" classname="map_prior" name="can_read_and_change_tau"/>
    <testcase time="1.16499999999999" classname="map_prior" name="can_read_and_change_hist_borrow"/>
    <testcase time="5.09500000000001" classname="map_prior" name="can_read_and_change_effective_sampling_method"/>
    <testcase time="1.15699999999998" classname="map_prior" name="menu_prior_is_invalidated_when_any_change_in_the_input_values_to_map_prior_happens"/>
    <testcase time="4.21800000000002" classname="map_prior" name="menu_prior_is_invalidated_when_any_change_in_the_input_values_to_map_prior_happens"/>
    <testcase time="3.87299999999999" classname="map_prior" name="menu_prior_is_invalidated_when_any_change_in_the_input_values_to_map_prior_happens"/>
    <testcase time="3.864" classname="map_prior" name="menu_prior_is_invalidated_when_any_change_in_the_input_values_to_map_prior_happens"/>
  </testsuite>
  <testsuite name="new_trial_analysis" timestamp="2024-08-08T19:24:47Z" hostname="f5d6db3fc801" tests="23" skipped="0" failures="1" errors="2" time="26.726">
    <testcase time="5.982" classname="new_trial_analysis" name="ae_div_side_and_main_is_hidden_when_analysis_type_is_BSAFE_CHOICES_SEL_ANALYSIS_1_"/>
    <testcase time="0.00600000000000023" classname="new_trial_analysis" name="ae_div_side_and_main_is_hidden_when_analysis_type_is_BSAFE_CHOICES_SEL_ANALYSIS_1_"/>
    <testcase time="0.00899999999998613" classname="new_trial_analysis" name="inci_div_side_and_main_is_visible_when_analysis_type_is_BSAFE_CHOICES_SEL_ANALYSIS_1_"/>
    <testcase time="0.00600000000000023" classname="new_trial_analysis" name="inci_div_side_and_main_is_visible_when_analysis_type_is_BSAFE_CHOICES_SEL_ANALYSIS_1_"/>
    <testcase time="1.798" classname="new_trial_analysis" name="can_read_and_change_number_of_patients"/>
    <testcase time="1.423" classname="new_trial_analysis" name="can_read_and_change_number_of_patients_with_AE"/>
    <testcase time="1.43900000000002" classname="new_trial_analysis" name="number_of_patients_with_AE_slider_is_updated_when_number_of_patients_change"/>
    <testcase time="1.40599999999998" classname="new_trial_analysis" name="number_of_patients_with_AE_slider_is_updated_when_number_of_patients_change"/>
    <testcase time="0.715000000000003" classname="new_trial_analysis" name="mix_density_plot_is_present"/>
    <testcase time="0.718999999999994" classname="new_trial_analysis" name="summary_table_is_present">
      <failure type="failure" message="Check on 'tbl' failed: Must be of type 'string', not 'list' ('test-new_trial_analysis.R:77:5')">Check on 'tbl' failed: Must be of type 'string', not 'list'
Backtrace:
    ▆
 1. └─checkmate::expect_string(tbl, fixed = "&lt;table") at test-new_trial_analysis.R:77:5
 2.   └─checkmate::makeExpectation(x, res, info, label)</failure>
    </testcase>
    <testcase time="0.733000000000004" classname="new_trial_analysis" name="new_trial_analysis_is_a_string"/>
    <testcase time="0.713999999999999" classname="new_trial_analysis" name="post_dist_is_a_string"/>
    <testcase time="0.712000000000018" classname="new_trial_analysis" name="current_trial_data_is_a_string"/>
    <testcase time="0.745000000000005" classname="new_trial_analysis" name="compare_plot_is_a_ggplot"/>
    <testcase time="0.72199999999998" classname="new_trial_analysis" name="map_summary_table_is_a_data.frame">
      <error type="error" message="Error in `bsafe::summary_stat_all_display(select_analysis = analysis_type(),  ('test-new_trial_analysis.R:101:5')">Error in `bsafe::summary_stat_all_display(select_analysis = analysis_type(), 
    robust_map_object = robust_map_mcmc(), ess_method = ess_method(), 
    current_trial_data = current_trial_data(), post_dist = post_dist(), 
    numerical = FALSE, seed = seed())`: unused arguments (numerical = FALSE, seed = seed())
Backtrace:
     ▆
  1. ├─shiny::isolate(app$get_values()[["export"]][[tns("r")]][["compare_summary_table"]]()) at test-new_trial_analysis.R:101:5
  2. │ ├─shiny::..stacktraceoff..(...)
  3. │ └─ctx$run(...)
  4. │   ├─promises::with_promise_domain(...)
  5. │   │ └─domain$wrapSync(expr)
  6. │   ├─shiny::withReactiveDomain(...)
  7. │   │ └─promises::with_promise_domain(...)
  8. │   │   └─domain$wrapSync(expr)
  9. │   │     └─base::force(expr)
 10. │   └─env$runWith(self, func)
 11. │     └─shiny (local) contextFunc()
 12. │       └─shiny::..stacktraceon..(expr)
 13. └─app$get_values()[["export"]][[tns("r")]][["compare_summary_table"]]()
 14.   ├─shinymeta:::metaDispatch(...)
 15.   │ ├─shinymeta:::switchMetaMode(...)
 16.   │ └─base::force(normal)
 17.   └─shiny (local) r_normal()</error>
    </testcase>
    <testcase time="0.961000000000013" classname="new_trial_analysis" name="compare_plot_code_eval_matches_plot"/>
    <testcase time="0.715000000000003" classname="new_trial_analysis" name="compare_summary_table_code_eval_matches_value">
      <error type="error" message="Error in `bsafe::summary_stat_all_display(select_analysis = analysis_type(),  ('test-new_trial_analysis.R:110:5')">Error in `bsafe::summary_stat_all_display(select_analysis = analysis_type(), 
    robust_map_object = robust_map_mcmc(), ess_method = ess_method(), 
    current_trial_data = current_trial_data(), post_dist = post_dist(), 
    numerical = FALSE, seed = seed())`: unused arguments (numerical = FALSE, seed = seed())
Backtrace:
     ▆
  1. └─teal.modules.bsafe:::expect_code_val_match(...) at test-new_trial_analysis.R:110:5
  2.   ├─shiny::isolate(export[["r"]][[n]]()) at tests/testthat/setup.R:22:3
  3.   │ ├─shiny::..stacktraceoff..(...)
  4.   │ └─ctx$run(...)
  5.   │   ├─promises::with_promise_domain(...)
  6.   │   │ └─domain$wrapSync(expr)
  7.   │   ├─shiny::withReactiveDomain(...)
  8.   │   │ └─promises::with_promise_domain(...)
  9.   │   │   └─domain$wrapSync(expr)
 10.   │   │     └─base::force(expr)
 11.   │   └─env$runWith(self, func)
 12.   │     └─shiny (local) contextFunc()
 13.   │       └─shiny::..stacktraceon..(expr)
 14.   └─export[["r"]][[n]]()
 15.     ├─shinymeta:::metaDispatch(...)
 16.     │ ├─shinymeta:::switchMetaMode(...)
 17.     │ └─base::force(normal)
 18.     └─shiny (local) r_normal()</error>
    </testcase>
    <testcase time="5.66199999999998" classname="new_trial_analysis" name="ae_div_side_and_main_is_visible_when_analysis_type_is_BSAFE_CHOICES_SEL_ANALYSIS_2_"/>
    <testcase time="0.007000000000005" classname="new_trial_analysis" name="ae_div_side_and_main_is_visible_when_analysis_type_is_BSAFE_CHOICES_SEL_ANALYSIS_2_"/>
    <testcase time="0.00800000000000978" classname="new_trial_analysis" name="inci_div_side_and_main_is_hidden_when_analysis_type_is_BSAFE_CHOICES_SEL_ANALYSIS_2_"/>
    <testcase time="0.00600000000000023" classname="new_trial_analysis" name="inci_div_side_and_main_is_hidden_when_analysis_type_is_BSAFE_CHOICES_SEL_ANALYSIS_2_"/>
    <testcase time="1.196" classname="new_trial_analysis" name="can_read_and_change_first_occurence"/>
    <testcase time="1.042" classname="new_trial_analysis" name="can_read_and_change_cummulative_time"/>
  </testsuite>
  <testsuite name="robust_map" timestamp="2024-08-08T19:25:08Z" hostname="f5d6db3fc801" tests="18" skipped="0" failures="1" errors="2" time="34.444">
    <testcase time="0.292000000000002" classname="robust_map" name="preface_rob_txt_returns_correct_preface_for_weakly_informative_conjugate_component"/>
    <testcase time="0.0029999999999859" classname="robust_map" name="preface_rob_txt_returns_correct_preface_for_EX_NEX_approach"/>
    <testcase time="0.00300000000001432" classname="robust_map" name="preface_rob_txt_returns_NULL_for_unknown_selection_analysis"/>
    <testcase time="5.77000000000001" classname="robust_map" name="robust_mean_div_is_hidden_when_analysis_type_is_BSAFE_CHOICES_SEL_ANALYSIS_1_"/>
    <testcase time="2.255" classname="robust_map" name="can_read_and_informative_prior_weight"/>
    <testcase time="5.703" classname="robust_map" name="can_read_and_change_ess_method"/>
    <testcase time="1.18599999999998" classname="robust_map" name="robust_map_mcmc_is_data.frame"/>
    <testcase time="1.20500000000001" classname="robust_map" name="robust_plot_is_ggplot"/>
    <testcase time="1.185" classname="robust_map" name="robust_summary_is_data.frame">
      <error type="error" message="Error in `bsafe::summary_stats_robust_map_prior_display(map_object = map_mcmc(),  ('test-robust_map.R:84:5')">Error in `bsafe::summary_stats_robust_map_prior_display(map_object = map_mcmc(), 
    select_analysis = analysis_type(), param_approx = param_approx(), 
    ess_method = ess_method(), robust_map_object = robust_map_mcmc(), 
    rob_ess_method = input[[BSAFE_ID$SEL_ROB_ESS_METHOD]], numerical = FALSE, 
    seed = seed())`: unused arguments (numerical = FALSE, seed = seed())
Backtrace:
     ▆
  1. ├─shiny::isolate(app$get_values()[["export"]][[tns("r")]][["robust_summary"]]()) at test-robust_map.R:84:5
  2. │ ├─shiny::..stacktraceoff..(...)
  3. │ └─ctx$run(...)
  4. │   ├─promises::with_promise_domain(...)
  5. │   │ └─domain$wrapSync(expr)
  6. │   ├─shiny::withReactiveDomain(...)
  7. │   │ └─promises::with_promise_domain(...)
  8. │   │   └─domain$wrapSync(expr)
  9. │   │     └─base::force(expr)
 10. │   └─env$runWith(self, func)
 11. │     └─shiny (local) contextFunc()
 12. │       └─shiny::..stacktraceon..(expr)
 13. └─app$get_values()[["export"]][[tns("r")]][["robust_summary"]]()
 14.   ├─shinymeta:::metaDispatch(...)
 15.   │ ├─shinymeta:::switchMetaMode(...)
 16.   │ └─base::force(normal)
 17.   └─shiny (local) r_normal()</error>
    </testcase>
    <testcase time="1.19299999999998" classname="robust_map" name="preface_text_is_present"/>
    <testcase time="1.22800000000001" classname="robust_map" name="density_function_is_present"/>
    <testcase time="1.19200000000001" classname="robust_map" name="robust_map_plot_is_present"/>
    <testcase time="1.227" classname="robust_map" name="table_is_present">
      <failure type="failure" message="Check on 'tbl' failed: Must be of type 'string', not 'list' ('test-robust_map.R:105:5')">Check on 'tbl' failed: Must be of type 'string', not 'list'
Backtrace:
    ▆
 1. └─checkmate::expect_string(tbl, fixed = "&lt;table") at test-robust_map.R:105:5
 2.   └─checkmate::makeExpectation(x, res, info, label)</failure>
    </testcase>
    <testcase time="1.46099999999998" classname="robust_map" name="robust_map_mcmc_code_eval_matches_value"/>
    <testcase time="1.44400000000002" classname="robust_map" name="robust_plot_code_eval_matches_value"/>
    <testcase time="1.196" classname="robust_map" name="robust_summary_code_eval_matches_value">
      <error type="error" message="Error in `bsafe::summary_stats_robust_map_prior_display(map_object = map_mcmc(),  ('test-robust_map.R:117:5')">Error in `bsafe::summary_stats_robust_map_prior_display(map_object = map_mcmc(), 
    select_analysis = analysis_type(), param_approx = param_approx(), 
    ess_method = ess_method(), robust_map_object = robust_map_mcmc(), 
    rob_ess_method = input[[BSAFE_ID$SEL_ROB_ESS_METHOD]], numerical = FALSE, 
    seed = seed())`: unused arguments (numerical = FALSE, seed = seed())
Backtrace:
     ▆
  1. └─teal.modules.bsafe:::expect_code_val_match(...) at test-robust_map.R:117:5
  2.   ├─shiny::isolate(export[["r"]][[n]]()) at tests/testthat/setup.R:22:3
  3.   │ ├─shiny::..stacktraceoff..(...)
  4.   │ └─ctx$run(...)
  5.   │   ├─promises::with_promise_domain(...)
  6.   │   │ └─domain$wrapSync(expr)
  7.   │   ├─shiny::withReactiveDomain(...)
  8.   │   │ └─promises::with_promise_domain(...)
  9.   │   │   └─domain$wrapSync(expr)
 10.   │   │     └─base::force(expr)
 11.   │   └─env$runWith(self, func)
 12.   │     └─shiny (local) contextFunc()
 13.   │       └─shiny::..stacktraceon..(expr)
 14.   └─export[["r"]][[n]]()
 15.     ├─shinymeta:::metaDispatch(...)
 16.     │ ├─shinymeta:::switchMetaMode(...)
 17.     │ └─base::force(normal)
 18.     └─shiny (local) r_normal()</error>
    </testcase>
    <testcase time="5.732" classname="robust_map" name="robust_mean_div_is_visible_when_analysis_type_is_BSAFE_CHOICES_SEL_ANALYSIS_2_"/>
    <testcase time="2.16899999999998" classname="robust_map" name="can_read_and_change_informative_prior_mean"/>
  </testsuite>
  <testsuite name="select_analysis" timestamp="2024-08-08T19:25:42Z" hostname="f5d6db3fc801" tests="16" skipped="0" failures="0" errors="0" time="7.897">
    <testcase time="0.302000000000021" classname="select_analysis" name="get_arm_choices_and_selection_returns_correct_choices_when_previous_selection_is_empty"/>
    <testcase time="0.00399999999999068" classname="select_analysis" name="get_arm_choices_and_selection_returns_correct_selected_value_when_previous_selection_is_empty"/>
    <testcase time="0.0029999999999859" classname="select_analysis" name="get_arm_choices_and_selection_returns_correct_choices_when_previous_selection_is_not_empty"/>
    <testcase time="0.00300000000001432" classname="select_analysis" name="get_arm_choices_and_selection_returns_correct_selected_value_when_previous_selection_is_not_empty"/>
    <testcase time="0.0029999999999859" classname="select_analysis" name="get_safety_topic_choices_and_selection_returns_correct_choices_when_there_are_choices"/>
    <testcase time="0.00300000000001432" classname="select_analysis" name="get_safety_topic_choices_and_selection_returns_correct_selected_value_when_there_are_choices"/>
    <testcase time="0.00399999999999068" classname="select_analysis" name="get_safety_topic_choices_and_selection_returns_correct_choices_when_previous_selection_is_not_empty"/>
    <testcase time="0.00300000000001432" classname="select_analysis" name="get_safety_topic_choices_and_selection_returns_correct_selected_value_when_previous_selection_is_not_empty"/>
    <testcase time="0.0029999999999859" classname="select_analysis" name="get_safety_topic_choices_and_selection_returns_NULL_choices_when_there_are_no_choices"/>
    <testcase time="0.00499999999999545" classname="select_analysis" name="get_safety_topic_choices_and_selection_returns_NULL_selected_value_when_there_are_no_choices"/>
    <testcase time="3.60500000000002" classname="select_analysis" name="can_read_and_change_treatment"/>
    <testcase time="0.941000000000003" classname="select_analysis" name="can_read_and_change_change_analysis_type"/>
    <testcase time="0.947999999999979" classname="select_analysis" name="can_read_and_change_safety_topic"/>
    <testcase time="0.900000000000006" classname="select_analysis" name="can_read_and_change_seed"/>
    <testcase time="0.39500000000001" classname="select_analysis" name="returned_data_is_a_data.frame"/>
    <testcase time="0.775000000000006" classname="select_analysis" name="exported_data_code_matches_output"/>
  </testsuite>
  <testsuite name="utils" timestamp="2024-08-08T19:25:50Z" hostname="f5d6db3fc801" tests="5" skipped="0" failures="0" errors="0" time="0.071">
    <testcase time="0.0589999999999975" classname="utils" name="_returns_a_when_a_is_not_NULL_not_NA_and_has_length_0"/>
    <testcase time="0.00300000000001432" classname="utils" name="_returns_b_when_a_is_NULL"/>
    <testcase time="0.0029999999999859" classname="utils" name="_returns_b_when_a_is_NA"/>
    <testcase time="0.00300000000001432" classname="utils" name="_returns_b_when_a_is_an_empty_vector"/>
    <testcase time="0.0029999999999859" classname="utils" name="_returns_b_when_a_is_a_vector_with_length_0"/>
  </testsuite>
</testsuites>
